apps/system/index.html

```diff
@@ -857,10 +857,9 @@ <h1 id="app-install-message"></h1>
             <dt data-l10n-id="size">Size</dt>
               <dd id="app-install-size"></dd>
             <dt data-l10n-id="author">Author</dt>
-              <dd>
-                <span id="app-install-author-name"></span><br />
-                <span id="app-install-author-url"></span>
-              </dd>
+              <dd id="app-install-author-name"></dd>
+            <dt>&nbsp;</dt>
```

Comment:
Your usage of ```<dd>``` and ```<dt>``` don't really make sense to me here. It was my understanding that each ```<dd>``` field is supposed to correlate with a ```<dt>``` field. I don't think this usage holds semantic value.

I would like to either be corrected, have the code revert to the original markup, or use some other, new markup.